did_peer/src/key/key.rs

```diff
@@ -0,0 +1,244 @@
+use crate::error::DidPeerError;
+
+use super::SupportedKeyType;
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct Key {
+    key_type: SupportedKeyType,
+    key: Vec<u8>,
```

Comment:
Hmm... If you don't like `key` (I can see why - it too generic), I would suggest `key_bytes`, for example?

Reply:
I was concerned that the accessing the Vec data would look awkward eg `key.key`, but just realized these are private, so that probably won't occur, as it will be rather accessed via `self`.
So that being said, I think am fine with `key`, or `key_bytes` as well